plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java

```diff
@@ -39,6 +39,7 @@
     private Duration maxBackoffDelay = RetryDriver.DEFAULT_SLEEP_TIME;
     private Duration maxRetryTime = RetryDriver.DEFAULT_MAX_RETRY_TIME;
     private boolean impersonationEnabled;
+    private boolean sparkStatisticsFallbackEnabled = true;
```

Comment:
Why `true`  by default?
Isn't this rather a fallback for very specific use-cases?

Reply:
This option is a safeguard if we don't have hive statistics in place, and statistics from spark are doing more harm than not having statistics at all.

Reply:
I think I didn't express my self clearly.
Do we really want by default to read spark stats in `trino-hive` when the Trino stats are missing?

Reply:
Yes, these are statistics generated by Spark SQL in it's hive catalog at https://github.com/apache/spark/blob/25133684ef9991d0b0414d97ac64c70e878b4d4d/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala#L1381
They are meant to be used for hive tables. The idea here is to utilise statistics generated by Spark in environments where Apache Spark is the ETL engine instead of Apache Hive (or Trino) and the hive catalog of Spark is being used to generate statistics.